jscl.lisp

```diff
@@ -58,6 +58,7 @@
 ;;; compiled in the host
 (defvar *source*
   '(("boot"          :target)
+    ("early-char"    :target)
```

Comment:
why early char?

are we actually calling any of this functions before char.lisp is loaded?

Note that the body of the macros are executed in the host, therefore the host version of the functions are used. Only :load-toplevel would force us to use early-char.lisp.

For example, if you use it in the value of a defvar, but it is not very common.

It is okay for early functions to call later functions, as long as the the early function is not called until later.

Anyway, I can also just try to put it together with char.lisp.